src/test/java/org/churchbooks/churchbooks/transactions/TransactionLoaderTest.java

```diff
@@ -0,0 +1,58 @@
+package org.churchbooks.churchbooks.transactions;
+
+import com.webcohesion.ofx4j.domain.data.common.Transaction;
+import com.webcohesion.ofx4j.io.OFXParseException;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class TransactionLoaderTest {
+    private final TransactionLoader loader = new TransactionLoader();
+
+
+    @Test
```

Comment:
Can we use @DisplayName attribute here to make it easy to understand what our tests are doing? Something like:

@DisplayName("When a valid file is provided the transactions are loaded successfully").

Let's also discuss in the meeting a convention we'll follow for test naming generally. Possibly what you have here or maybe something different.